### DISCUSSION TITLE
Notification Sounds in DAB / Music Collection

### DISCUSSION THREAD

**Discussion by Fatbloke72** [BimmerGestalt/AAIdrive#650] [Q&A]
Hi, loving the app and notifications and Spotify are working great. Two quick questions though.

1. I have notifications set up a d they display perfectly. However I didn't receive any notification Sounds when listening to DAB, CD, or music collection. Is there a way to force throughtbther later sounds either in the car (above the radio for example), or maintaining the notification sound on my Samsung s21?

2. With regards Spotify, do I correctly assume that I can't access downloadedbplaylists etc when Spotify is in 'offline mode' ?

Thank you 👍🏻

**Comment by hufman** [comment]
Thanks for the compliments!
Unfortunately, I don't know of a way to temporarily claim audio in the car to play a notification sound and then release audio to resume DAB or other sources.
AAIdrive only shows what the music app returns in its browse results, and it unfortunately seems that Spotify chooses to not return any browse results when offline.
